cocos/core/splash-screen-webgl.ts

```diff
@@ -221,30 +223,45 @@ export class SplashScreenWebgl {
     private init () {
         // adapt for native mac & ios
         if (JSB) {
-            if (sys.os == cc.sys.OS_OSX || sys.os == cc.sys.OS_IOS) {
+            if (sys.os == sys.OS_OSX || sys.os == sys.OS_IOS) {
                 this.gl.canvas.width = screen.width * devicePixelRatio;
                 this.gl.canvas.height = screen.height * devicePixelRatio;
             }
         }
 
-        // TODO: hack for cocosPlay & XIAOMI cause on landscape canvas value is wrong
+        // adapt for cocos play 1.0.11
+        if (COCOSPLAY) {
```

Comment:
There is no version distinguish between this if and the next one, it's not acceptable to run the lame adaptation code  twice for the same platform.